app/assets/javascripts/active_admin/lib/batch_actions.es6

```diff
@@ -1,9 +1,13 @@
 const onDOMReady = function() {
+  // Detach any previously attached handlers before re-attaching them.
+  // This avoids double-registered handlers when Turbolinks is enabled
+  $('.batch_actions_selector li a').off();
```

Comment:
I think it may be better to remove only the events that ActiveAdmin uses in the case that a user has set their own. I'd suggest trying this:

```suggestion
  $('.batch_actions_selector li a').off('click confirm:complete');
```

That should work in our case based on the jQuery docs.
> events
> Type: String
> One or more space-separated event types and optional namespaces, or just namespaces, such as "click", "keydown.myPlugin", or ".myPlugin".